### PR TITLE
Feature/qbiossdk 1118

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Pakhimoyn** [comment]
letonhossin@gmail.com

File: QMChatViewController/QMChatDataSource/QMChatDataSource.m [hunk 1]
```diff
@@ -204,7 +236,7 @@ - (void)calDelegateMethodForIndexPaths:(NSArray *)indexPaths withUpdateType:(QMD
 
 - (NSArray *)allMessages {
     
-    return [self.messages copy];
```
Comment (mullayaasin) [inline QMChatViewController/QMChatDataSource/QMChatDataSource.m:207]: all

Comment (Raikerian) [reply QMChatViewController/QMChatDataSource/QMChatDataSource.m:207]: @mullayaasin Hi there, what do you mean?

Comment (mullayaasin) [reply QMChatViewController/QMChatDataSource/QMChatDataSource.m:207]: This diff

Comment (Raikerian) [reply QMChatViewController/QMChatDataSource/QMChatDataSource.m:207]: @mullayaasin please specify the exact issue with it

**Pull request merged**